ext/nnstreamer/tensor_decoder/tensordec-boundingbox.cc

```diff
@@ -329,11 +339,114 @@ iou (detectedObject *a, detectedObject *b)
   return (o >= 0) ? o : 0;
 }
 
+/**
+ * @brief Calculate the corners using center position and angle
+ */
+void
+get_rotated_rect_corners (detectedObject *obj, Point corners[4])
+{
+  float cx = obj->x;
+  float cy = obj->y;
+  float angle = obj->angle;
+
+  // if angle is degree
+  /*
+  float angle_rad = angle * (M_PI / 180.0f);
+  float cos_a = cos(angle_rad);
+  float sin_a = sin(angle_rad);
+  */
```

Comment:
Commented out codes only confuse other developers if it is not about "how to use this function".